packages/react/src/Button/Button.tsx

```diff
@@ -16,6 +16,7 @@ export type ButtonBaseProps = {
   variant?: typeof ButtonVariants[number]
   size?: typeof ButtonSizes[number]
   hasArrow?: boolean
+  ariaDisabled?: boolean
```

Comment:
Given that Button is a wrapper around HTMLElement, could it be confusing that we're redeclaring this rather than relying on the inherited attribute?

Reply:
That makes sense. I've adjusted it so that we rely on `aria-disabled` instead!